### ISSUE TITLE
Images no longer loading after "a while"

### ISSUE THREAD

**Issue by chevdor** [hay-kot/homebox#897]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### Homebox Version

v0.10.3

### What is the issue you are experiencing?

After "a while" (which I think is when the login cookie expires or something alike), the images are no longer displayed.
Here is a screenshot of a search with the search file empty:
<img width="772" alt="image" src="https://github.com/hay-kot/homebox/assets/738724/ff2b6609-560e-4fa3-bd51-4ebece4b368d">

If I sign out then sign in again (although I am using "remember me"), the issue is fixed.
<img width="769" alt="image" src="https://github.com/hay-kot/homebox/assets/738724/46b3c5e0-dec2-417a-ade9-45f1d90497a9">

It sounds like my  session does expirre although I used "remember me". If the session did nonetheless expire, the question is why I can still browse and search?

I am using Brave but with shields down for Homebox.

### How can the maintainer reproduce the issue?

I guess you would need to expirer those cookies and see what happends.

### Deployment

Other

### Deployment Details

Docker / K8S.

**Comment by Spreadcat** [comment]
Not sure how to solve this permanently, but if somebody else encounters this bug, a temporary solution is to logout and re-login into homebox.

**Comment by chevdor** [comment]
Yes, I confirm that logout/login is a workaround. Yet it seems a very repeatable.
Happy to provide more info on the env if that helps. Do you happen to have a link to the other issue ?

**Comment by Spreadcat** [comment]
Other issue? Not sure I get what you mean?

**Comment by Spreadcat** [comment]
Other issue? Not sure I get what you mean?
